i18n/ru/docusaurus-plugin-content-docs/current/guides/migration/from-v1.md

```diff
@@ -59,7 +59,7 @@ sidebar_position: 4
 - будет в целом больше документации для онбординга новых людей и изучения концепций методологии
 - будет развиваться в дальнейшем тулкит для соблюдения концепций и конвенций по архитектуре
 
-> Само собой, будет поддержка пользователей и по первой версии - но для нас первоочередна все же последняя версия
+> Само собой, будет поддержка пользователей и по первой версии - но для нас первоочередная все же последняя версия
```

Comment:
issue: I don't really agree with this change, I think the old variant sounds more cohesive. If you have resources that you referenced to make this change, I'd be glad to take a look

Reply:
The feeling that there is no such word at all

Reply:
Okay, I looked it up, I think you're right